src/m_misc.c

```diff
@@ -817,70 +817,70 @@ default_t defaults[] = {
   { // killough 3/1/98
     "monsters_remember",
     (config_t *) &default_monsters_remember, (config_t *) &monsters_remember,
-    {1}, {0,1}, number, ss_enem, wad_yes,
+    {1}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters remembering enemies after killing others"
   },
 
   { // killough 7/19/98
     "monster_infighting",
     (config_t *) &default_monster_infighting, (config_t *) &monster_infighting,
-    {1}, {0,1}, number, ss_enem, wad_yes,
+    {1}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters fighting against each other when provoked"
   },
 
   { // killough 9/8/98
     "monster_backing",
     (config_t *) &default_monster_backing, (config_t *) &monster_backing,
-    {0}, {0,1}, number, ss_enem, wad_yes,
+    {0}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters backing away from targets"
   },
 
   { //killough 9/9/98:
     "monster_avoid_hazards",
     (config_t *) &default_monster_avoid_hazards, (config_t *) &monster_avoid_hazards,
-    {1}, {0,1}, number, ss_enem, wad_yes,
+    {1}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters to intelligently avoid hazards"
   },
 
   {
     "monkeys",
     (config_t *) &default_monkeys, (config_t *) &monkeys,
-    {0}, {0,1}, number, ss_enem, wad_yes,
+    {0}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters to move up/down steep stairs"
   },
 
   { //killough 9/9/98:
     "monster_friction",
     (config_t *) &default_monster_friction, (config_t *) &monster_friction,
-    {1}, {0,1}, number, ss_enem, wad_yes,
+    {1}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters to be affected by friction"
   },
 
   { //killough 9/9/98:
     "help_friends",
     (config_t *) &default_help_friends, (config_t *) &help_friends,
-    {0}, {0,1}, number, ss_enem, wad_yes,
+    {0}, {0,1}, number, ss_none, wad_yes,
     "1 to enable monsters to help dying friends"
   },
 
   { // killough 7/19/98
     "player_helpers",
     (config_t *) &default_dogs, (config_t *) &dogs,
-    {0}, {0,3}, number, ss_enem, wad_yes,
+    {0}, {0,3}, number, ss_none, wad_yes,
     "number of single-player helpers"
```

Comment:
What if we put this back to the Enemies menu? Port named Woof, but there are no dogs anywhere 😄